x-pack/plugins/security_solution/public/threat_intelligence/routes.tsx

```diff
@@ -44,6 +48,15 @@ const ThreatIntelligence = memo(() => {
     sourcererDataView: sourcererDataView as unknown as SourcererDataView,
     getUseInvestigateInTimeline: useInvestigateInTimeline,
 
+    blockList: {
+      exceptionListApiClient: BlocklistsApiClient.getInstance(http),
+      useSetUrlParams,
+      // @ts-ignore
```

Comment:
I was struggling because the I can't use the [ArtifactFormComponentProps](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/management/components/artifact_list_page/types.ts#L25) and [ArtifactFlyoutProps](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/artifact_flyout.tsx#L154) types within our Threat Intelligence plugin because of circular dependancies.
Therefore I created separate interfaces ([here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/artifact_flyout.tsx#L154)).

I'm not sure how to do the type check properly from within the Security Solution folder. Any idea?